packages/x-tailwindcss/src/types.ts

```diff
@@ -1,9 +1,11 @@
 import { DeepPartial, Dictionary } from '@empathyco/x-utils';
 import { PluginAPI } from 'tailwindcss/types/config';
+import { Config } from 'tailwindcss';
```

Comment:
We could import both from `tailwindcss/types/config`, or it was intended to import it separately? 
```suggestion
import { PluginAPI, Config } from 'tailwindcss/types/config';
```

Reply:
the `PluginAPI` is not "publicly" exported by tailwind, I prefer to keep it separately if the path or type change in the future. And as the Config is publicly exported, we can use it from the default barrel, and we know it will not change, unless it is a breaking change.